omnikinverter/tcp.py

```diff
@@ -29,18 +31,20 @@ class _AcOutput(BigEndianStructure):
         ("power", c_ushort),
     ]
 
-    def get_power(self) -> Optional[int]:
+    def get_power(self) -> int | None:
         """Retrieve AC power.
 
-        Returns:
+        Returns
+        -------
             The power field, or None if it is unset.
         """
         return None if self.power == UINT16_MAX else self.power
 
-    def get_frequency(self) -> Optional[float]:
+    def get_frequency(self) -> float | None:
         """Retrieve AC frequency.
 
-        Returns:
+        Returns
+        -------
```

Comment:
You're keeping the colon below?

Reply:
Here you replaced `Returns:` with `Returns` (removing the colon `:`), but not in other places. Same for `Args:`, `Raises:` etc.

Reply:
Ahh so, yes that's something ruff does automatically, no idea why that doesn't happen consistently. If you adjust it then ruff will correct it again so there's not much you can do about it.

Reply:
Oh that's buggy :(